content/Video_Display.md

```diff
@@ -553,27 +553,28 @@ the one at `0:9800`!
 ### Background (BG)
 
 The [SCY and SCX](#ff42-scy-scroll-y-r-w-ff43-scx-scroll-x-r-w) registers can be
-used to scroll the background, allowing to select the origin of the visible
-160x144 pixel area within the total 256x256 pixel background map.
+used to scroll the Background, specifying the origin of the visible
```

Comment:
Maybe we should explicitly mention that the origin is the top-left on-screen pixel?

Reply:
Fixed